package.json

```diff
@@ -26,7 +26,7 @@
     "cf-expandables": "latest",
     "cf-forms": "latest",
     "cf-grid": "latest",
-    "cf-icons": "latest",
+    "cf-icons": "4.4.0",
```

Comment:
Should this be 4.4.1 now?

Reply:
Also, is this pinned and the other cf dependencies unpinned for a specific reason?

Reply:
Blerg, this is something npm does, and I almost always forget to reset it. Thanks for catching it.

Reply:
@wpears fixed, thanks again for noticing this.